task/init/0.2/MIGRATION.md

```diff
@@ -0,0 +1,10 @@
+# Migration from 0.1 to 0.2
+
+The parameter `skip-optional` used by `init` task was removed
+
+## Action from users
+
+Update files in Pull-Request created by RHTAP bot:
+- Search for the task named `init`
+- Remove the `skip-optional` parameter from the `specs.params` and from the `specs.steps.env` sections in the task
```

Comment:
User should not need to update the task directly. `skip-optional` param can be removed from generated PipelineRuns under `.tekton/`. The task is referenced via bundle resolver.

Reply:
I don't have any experience with creating new versions of the tasks/migration. I was basing my approach on the way things were handled in `deprecated-image-check` e.g https://github.com/redhat-appstudio/build-definitions/blob/main/task/deprecated-image-check/0.2/MIGRATION.md

@mmorhun Could you please share your opinion here?

Reply:
The users have [this](https://github.com/redhat-appstudio/image-controller/blob/889c0b1d674feb10123817a4270d0b4cfe39636e/.tekton/image-controller-pull-request.yaml#L142-L156) in their repo, so they should only need to remove deprecated params from the `params` section

Reply:
I see

Reply:
Removed all parameteres and the result and updated the migration doc